develop/reference/articles/xx-theme-file-reference/00-theme-reference-intro.markdown

```diff
@@ -0,0 +1,241 @@
+have a slightly different anatomy, the core theme files are the same. Note that 
+the `build` directory is shown for reference, and is not created by default.
+
+- `theme-name/`
+    - `build/`(not created automatically)
+        - `css/`
+            - [`_application.scss`](#application-scss)
+            - [`_aui_custom.scss`](#auicustom-scss)
+            - [`_aui_variables.scss`](#auivariables-scss)
+            - [`_base.scss`](#base-scss)
+            - [`_custom.scss`](#custom-scss)
+            - [`_extras.scss`](#extras-scss)
+            - [`_imports.scss`](#imports-scss)
+            - [`_layout.scss`](#layout-scss)
+            - [`_liferay_custom.scss`](#liferaycustom-scss)
+            - [`_liferay_variables_custom.scss`](#liferayvariablescustom-scss)
+            - [`_liferay_variables.scss`](#liferayvariables-scss)
+            - [`_navigation.scss`](#navigation-scss)
+            - [`_portal.scss`](#portal-scss)
+            - [`_portlet.scss`](#portlet-scss)
+            - [`_taglib.scss`](#taglib-scss)
+            - `application/`
+            - `aui/`
+            - [`aui.scss`](#aui-scss)
+            - `base/`
+            - `layout/`
+            - [`main.scss`](#main-scss)
+            - `navigation/`
+            - `portal/`
+            - `portlet/`
+            - `taglib/`
+        - `templates/`
+            - [`init_custom.ftl`](#initcustom-ftl)
+            - [`init.ftl`](#init-ftl)
+            - [`navigation.ftl`](#navigation-ftl)
+            - [`portal_normal.ftl`](#portalnormal-ftl)
+            - [`portal_pop_up.ftl`](#portalpopup-ftl)
+            - [`portlet.ftl`](#portlet-ftl)
+            - (other directories that have been copied from src)
+    - `dist/` (not created automatically)
+    - [`gulpfile.js`](#gulpfile-js)
+    - [`liferay-theme.json`](#liferay-theme-json)
+    - `node_modules/`
+        - (many directories)
+    - [`package.json`](#package-json)
+    - `src/`
+        - `css/`
+            - (modified CSS files)
+        - `images/`
+            -   (many directories)
+        - `js/`
+            - [`main.js`](#main-js)
+        - `templates/`
+            - (Modified theme templates)
+        - `WEB-INF/`
+                - `lib/`
+            - [`liferay-look-and-feel.xml`](#liferay-look-and-feel-xml)
+            - [`liferay-plugin-package.properties`](#liferay-plugin-package-properties)
+            - `src/`
+                - `resources-importer/`
+                    - (Many directories)
+                    
+Regarding CSS files, it is recommended that you only modify 
+`_custom.scss`, `_aui_custom.scss`, `_aui_variables.scss`, and 
+`_liferay_variables_custom.scss`.
+
+You can of course overwrite any CSS file that you wish, but if you modify any 
+other files, you will most likely be removing styling that @product@ 7 needs to 
+work properly. It's recommended that you overwrite the individual files that 
+aren't in the root `css` directory instead, i.e. 
```

Comment:
This might be a little confusing because the files that they _should_ overwrite (_custom.scss for example) are in the root `css` directory.